app/Http/Controllers/UserController.php

```diff
@@ -96,12 +102,18 @@ public function update(UserUpdateRequest $request, $id)
 
             return response()->json([
                 'error' => false,
-                'messageSuccess' => 'This user successfully updated',
+                'messageSuccess' => [
+                    'status' => 'Update success',
+                    'message' => 'This user successfully updated'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 107</br>Each line in an array declaration must end in a comma